### PR TITLE
Better logging and metrics for incoming gossip blocks and sent aggregate attestations

### PR DESCRIPTION
**Pull request by zah**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,7 +148,8 @@ make NIMFLAGS="-d:insecure" medalla
 You can now see the raw metrics on http://127.0.0.1:8008/metrics but they're not very useful like this, so let's feed them to a Prometheus instance:
 
 ```bash
-prometheus --config.file=build/data/shared_medalla_0/prometheus.yml
+cd build/data/shared_medalla_0
+prometheus --config.file=./prometheus.yml --storage.tsdb.path=./prometheus
```
Comment (stefantalpalaru) [inline README.md:152]: `--config.file=./prometheus.yml` is the default
What's wrong with the default of "data" for it's db directory? Too generic?

Comment (stefantalpalaru) [reply README.md:152]: Oh, another benefit of keeping the Prometheus db outside that "build/data/shared..." directory is that you can keep your Prometheus daemon running, delete that node data dir and restart your node, catching all the metrics from start to finish.

Comment (zah) [reply README.md:152]: For me, the default was not `./data`. Without this option Prometheus tries to write to a system directory that requires escalated rights.

Comment (zah) [reply README.md:152]: Otherwise, I agree that we can have a separate directory.

Comment (stefantalpalaru) [reply README.md:152]: Probably different versions. I'm using Prometheus-2.19.1 and this is in my `--help` output:

```text
--storage.tsdb.path="data/"  
                                 Base path for metrics storage.
```

**Pull request merged**